schemas/lakehouse/aep-lakehouse-batch-errors.schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/schemas/aep-lakehouse-batch-errors",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AEP Lakehouse Batch Errors",
+  "description": "Global schema for AEP Lakehouse datasets containing batch errors for diagnostics.",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "definitions": {
+    "batchInputFiles": {
+      "properties": {
+        "xdm:datasetId": {
```

Comment:
Following XDM's coding style, pl. change the `Id` to `ID` part in all occurrences.